### PR TITLE
ISPN-5863 & ISPN-5877

### PR DESCRIPTION
**Pull request by galderz**
https://issues.jboss.org/browse/ISPN-5863
https://issues.jboss.org/browse/ISPN-5877

### CODE REVIEW

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SkipCacheLoadFlagTest.java [hunk 1]
```diff
@@ -92,6 +94,11 @@ public void testContainsKey() {
       performTest(RequestType.CONTAINS);
    }
 
+   public void testPutAll() {
+      //ContainsKeyRequest
```
Comment (danberindei) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SkipCacheLoadFlagTest.java]: Comment is wrong, and TBH it doesn't seem necessary.

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SkipIndexingFlagTest.java [hunk 2]
```diff
@@ -93,6 +95,11 @@ public void testContainsKey() {
       performTest(RequestType.CONTAINS);
    }
 
+   public void testPutAll() {
+      //ContainsKeyRequest
```
Comment (danberindei) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/SkipIndexingFlagTest.java]: Here as well, the comment is wrong.

File: core/src/main/java/org/infinispan/interceptors/InterceptorChain.java [hunk 3]
```diff
@@ -333,7 +333,7 @@ public void appendInterceptor(CommandInterceptor ci, boolean isCustom) {
     */
    public Object invoke(InvocationContext ctx, VisitableCommand command) {
       try {
-         return command.acceptVisitor(ctx, firstInChain);
+            return command.acceptVisitor(ctx, firstInChain);
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/interceptors/InterceptorChain.java]: Unnecessary whitespace change.

**Comment by galderz** [comment]
@danberindei Fixed

**Comment by wburns** [comment]
Seems good to me now, pulling...

**Comment by wburns** [comment]
Integrated into master, thanks @galderz !  Do we want 8.0.x or anything?
